ethereumetl/cli/stream.py

```diff
@@ -68,7 +71,7 @@ def stream(last_synced_block_file, lag, provider_uri, output, start_block, entit
 
     streamer_adapter = EthStreamerAdapter(
         batch_web3_provider=ThreadLocalProxy(lambda: get_provider_from_uri(provider_uri, batch=True)),
-        item_exporter=create_item_exporters(output),
+        item_exporter=create_item_exporter(output, connection_url),
```

Comment:
Should be `create_item_exporters` - notice `s` at the end

Reply:
Pushed, Thanks